src/main/java/ledger/database/storage/ISQLiteDatabaseTransaction.java

```diff
@@ -187,18 +202,18 @@ default void editTransaction(Transaction transaction) throws StorageException {
     default List<Transaction> getAllTransactions() throws StorageException {
         try {
             Statement stmt = getDatabase().createStatement();
-            ResultSet rs = stmt.executeQuery("SELECT * FROM TRANSACT;");
+            ResultSet rs = stmt.executeQuery("SELECT * FROM " + TransactionTable.TABLE_NAME + ";");
```

Comment:
I got rid of all occurrences of "SELECT *" in the ISQLiteDatabase____ files in my refactoring work on the f_2refactoring branch. These changes might not merge nicely, though. (Applies to all other "SELECT *" change requests, I believe)